### PR TITLE
Collect more analytics

### PR DESCRIPTION
**Pull request by irlaec**

### CODE REVIEW

**Comment by irlaec** [comment]
[Analytics: Collect more values](https://trello.com/c/XngFZw4y/176-analytics-collect-more-values)

**Comment by irlaec** [comment]
[Analytics: Collect more values](https://trello.com/c/XngFZw4y/176-analytics-collect-more-values)

**Comment by irlaec** [comment]
[Analytics: Collect more values](https://trello.com/c/XngFZw4y/176-analytics-collect-more-values)

**Comment by irlaec** [comment]
[Analytics: Collect more values](https://trello.com/c/XngFZw4y/176-analytics-collect-more-values)

**Comment by irlaec** [comment]
[Analytics: Collect more values](https://trello.com/c/XngFZw4y/176-analytics-collect-more-values)

**Comment by irlaec** [comment]
[Analytics: Collect more values](https://trello.com/c/XngFZw4y/176-analytics-collect-more-values)

**Comment by irlaec** [comment]
[Analytics: Collect more values](https://trello.com/c/XngFZw4y/176-analytics-collect-more-values)

**Comment by irlaec** [comment]
[Analytics: Collect more values](https://trello.com/c/XngFZw4y/176-analytics-collect-more-values)

**Comment by irlaec** [comment]
[Analytics: Collect more values](https://trello.com/c/XngFZw4y/176-analytics-collect-more-values)

File: octoprint_mrbeam/static/js/analytics.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ $(function () {
         window.mrbeam.viewModels['analyticsViewModel'] = self;
 
         self.send_fontend_event = function (event, payload) {
-            payload['ts'] = payload['ts'] || new Date().getTime();
+            payload['ts'] = payload['ts'] || new Date().getTime()/1000;
```
Comment (andywerner) [inline octoprint_mrbeam/static/js/analytics.js]: Do we already have frontend analytics released? if yes, you'll end up with two different ts formats in the backend... Let's talk about this one.

File: octoprint_mrbeam/static/js/wizard_general.js [hunk 2]
```diff
@@ -18,40 +22,51 @@ $(function() {
...
+                });
+            }
         };
 
         self.onAllBound = function () {
-            self.uuid(self.settings.settings.plugins.findmymrbeam.uuid());
-            self.registered(self.settings.settings.plugins.findmymrbeam.registered());
-            self.ping(self.settings.settings.plugins.findmymrbeam.ping());
+            if (self.isWhatsnew) {
+                self.uuid(self.settings.settings.plugins.findmymrbeam.uuid());
```
Comment (andywerner) [inline octoprint_mrbeam/static/js/wizard_general.js:47]: Can you be sure, that settings are already loaded in onAllBound?

Comment (andywerner) [reply octoprint_mrbeam/static/js/wizard_general.js:47]: I asked Gina since I'm never sure.

Comment (andywerner) [reply octoprint_mrbeam/static/js/wizard_general.js:47]: ok, it find. data are there. http://docs.octoprint.org/en/master/plugins/viewmodels.html#web-interface-startup

**Review by andywerner — CHANGES_REQUESTED** [review]
timestamps

File: octoprint_mrbeam/static/js/analytics.js (hunk 1 above)
Comment (irlaec) [reply octoprint_mrbeam/static/js/analytics.js]: I left it as it was.

File: octoprint_mrbeam/static/js/convert.js [hunk 3]
```diff
@@ -814,20 +814,35 @@ $(function(){
...
                 let format = typePath[typePath.length - 1];
 
-                let sub_format;
+                let sub_format, font, text_length;
                 if (format === "image") {
                     let file_name = $('#' + currentDesign.id).find('.title').text();
                     sub_format = file_name.split('.').pop(-1).toLowerCase();
                 }
 
+                console.log('#######################');
```
Comment (andywerner) [inline octoprint_mrbeam/static/js/convert.js]:
```suggestion
```

File: octoprint_mrbeam/static/js/convert.js [hunk 4]
```diff
@@ -814,20 +814,35 @@ $(function(){
...
+                console.log(format);
+                if (format === 'quicktext') {
+                    console.log(currentDesign);
```
Comment (andywerner) [inline octoprint_mrbeam/static/js/convert.js]: pls remove log lines

**Review by andywerner — APPROVED** [review]

File: octoprint_mrbeam/static/js/convert.js (hunk 4 above)
Comment (irlaec) [reply octoprint_mrbeam/static/js/convert.js]: removed

File: octoprint_mrbeam/static/js/convert.js (hunk 3 above)
Comment (irlaec) [reply octoprint_mrbeam/static/js/convert.js]: removed

File: octoprint_mrbeam/static/js/convert.js [hunk 5]
```diff
@@ -814,35 +814,35 @@ $(function(){
...
                 if (format === "image") {
                     let file_name = $('#' + currentDesign.id).find('.title').text();
                     sub_format = file_name.split('.').pop(-1).toLowerCase();
                 }
 
-                console.log('#######################');
-                console.log(format);
+                if (format === 'svg') {
+                    clip_working_area = self.workingArea.gc_options().clip_working_area;
+                    console.log(clip_working_area)
```
Comment (andywerner) [inline octoprint_mrbeam/static/js/convert.js]:
```suggestion
```

**Review by andywerner — CHANGES_REQUESTED** [review]

**Pull request merged**